org.scala-ide.sdt.core/src/org/scalaide/core/internal/project/ScalaProject.scala

```diff
@@ -582,10 +585,11 @@ class ScalaProject private(val underlying: IProject) extends ClasspathManagement
       logger.info("BM: SBT enhanced Build Manager for " + IScalaPlugin().scalaVersion + " Scala library")
```

Comment:
this logging should be replaced with something that tells which build manager is used.